content/cloud/cloud-command-reference.md

```diff
@@ -1158,6 +1289,63 @@ OPTIONS:
 
 {{ blockEnd }}
 
+{{ startTab "v0.7.0"}}
+
+Spin compatibility: `>= v1.3`
+
+<!-- @selectiveCpy -->
+```console
+$ spin cloud deploy --help
+
+Package and upload an application to the Fermyon Cloud
+
+USAGE:
+    spin cloud deploy [OPTIONS]
+
+OPTIONS:
+        --build
+            For local apps, specifies to perform `spin build` before deploying the application.
+            
+            This is ignored on remote applications, as they are already built.
+            
+            [env: SPIN_ALWAYS_BUILD=]
+
+    -f, --from <APPLICATION>
+            The application to deploy. This may be a manifest (spin.toml) file, a directory
+            containing a spin.toml file, or a remote registry reference. If omitted, it defaults to
+            "spin.toml"
+
+    -h, --help
+            Print help information
+
+        --key-value <KEY_VALUES>
+            Set a key/value pair (key=value) in the deployed application's default store. Any
+            existing value will be overwritten. Can be used multiple times
+
+        --link <LINKS>
+            Specifies how application labels (such as SQLite databases) should be linked if they are
+            not already linked. This is intended for non-interactive environments such as release
+            pipelines; therefore, if any links are specified, all links must be specified.
+            
+            Links must be of the form 'sqlite:label=database'. Databases that do not exist will be
+            created.
```

Comment:
This made me realize this needed to be updated: https://github.com/fermyon/cloud-plugin/pull/176
```suggestion
        --link <LINKS>
            Specifies how application labels (such as SQLite databases) should be linked if they are
            not already linked. This is intended for non-interactive environments such as release
            pipelines; therefore, if any links are specified, all links must be specified.
            
            Links must be of the form 'sqlite:label=database' or 'kv:label=store'. Databases or key
            value stores that do not exist will be created.
```

Reply:
Thanks @Kate, I have updated the Cloud CLI again and re-run these commands to get the same output as above.
These changes will go in the next commit/push to this branch.